pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>3.0.6</version>
+  <version>3.1.0-alpha-1</version>
```

Comment:
We shouldn't be modifying the version like this. Please review the release notes in this project.

Reply:
👍